src/test/scala/higherkindness/compendium/core/ProtocolUtilsSpec.scala

```diff
@@ -33,30 +33,57 @@ object ProtocolUtilsSpec extends Specification with ScalaCheck {
   private val validator = ProtocolUtils.impl[IO]
 
   "Protocol validator" >> {
-    "Given a raw protocol text returns the same protocol if it validates correctly" >> {
+    "[Avro] Given a raw protocol text returns the same protocol if it validates correctly" >> {
       val stream: InputStream = getClass.getResourceAsStream("/correct.avro")
       val text: String        = scala.io.Source.fromInputStream(stream).getLines.mkString
       val protocol: Protocol  = Protocol(text)
 
-      validator.validateProtocol(protocol).unsafeRunSync === protocol
+      validator.validateProtocol(protocol, IdlName.Avro).unsafeRunSync === protocol
     }
 
-    "Given a raw protocol text raises an error if the protocol is incorrect" >> prop {
+    "[Avro] Given a raw protocol text raises an error if the protocol is incorrect" >> prop {
       protocol: Protocol =>
         validator
-          .validateProtocol(protocol)
+          .validateProtocol(protocol, IdlName.Avro)
           .unsafeRunSync must throwA[SchemaParseException]
     }
 
-    "Given multiple protocols validates them sequentially" >> {
+    "[Avro] Given multiple protocols validates them sequentially" >> {
       val stream: InputStream = getClass.getResourceAsStream("/correct.avro")
       val text: String        = scala.io.Source.fromInputStream(stream).getLines.mkString
       val protocol: Protocol  = Protocol(text)
 
-      val validation = validator.validateProtocol(protocol) *> validator.validateProtocol(protocol)
+      val validation = validator.validateProtocol(protocol, IdlName.Avro) *> validator
+        .validateProtocol(protocol, IdlName.Avro)
 
       validation.unsafeRunSync should not(throwA[SchemaParseException])
     }
+
+    "[Proto] Given a raw protocol text returns the same protocol if it validates correctly" >> {
+      val stream: InputStream = getClass.getResourceAsStream("/correct.proto")
+      val text: String        = scala.io.Source.fromInputStream(stream).getLines.mkString
+      val protocol: Protocol  = Protocol(text)
+      validator.validateProtocol(protocol, IdlName.Protobuf).unsafeRunSync === protocol
```

Comment:
is this enough to assert the validation works? shouldn't be `mustBe`?

Reply:
🤔 I checked the documentation, `===` is the same as `mustEqual`  and `must be_` 
https://etorreborre.github.io/specs2/guide/SPECS2-3.4/org.specs2.guide.Matchers.html